src/com/facebook/buck/jvm/java/abi/source/InterfaceValidator.java

```diff
@@ -139,8 +136,7 @@ private boolean isCompiledInCurrentRun(TypeElement typeElement) {
             }
 
             private boolean isOnBootClasspath(TypeElement typeElement) {
-              return bootClasspathOracle.isOnBootClasspath(
-                  elements.getBinaryName(typeElement).toString());
+              return bootClasspathOracle.isOnBootClasspath(typeElement.getQualifiedName().toString());
```

Comment:
This is not the same as the binary name (https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.1), so you'd need some additional logic to translate it appropriately.